src/utils/barChart.utils.ts

```diff
@@ -27,7 +27,7 @@ export function convertToBarSeries(items: MultiSeriesDataItems[], labelsMap: Lab
     const label = item.label_key.toString();
 
     const result: BarDataMap = {
-      [BAR_CHART_X_LABEL]: getTranslatedLabel(label, labelsMap)
+      [BAR_CHART_X_LABEL]: getTranslatedLabel(label, labelsMap),
     };
 
     if (!item.series) {
```

Comment:
add here the logic from your function `convertItemsToBarSeries` in order to  distinguish where a single bar chart doesn't have a label Map.

Reply:
I've changed the labelsMap type to any, it seems to work fine now. Is that ok or we need a specific type?